operator-wg/whitepaper/README.md

```diff
@@ -43,4 +42,8 @@
 - [ ] April 20th - April 30th, 2021 - *Final group review*
 * Final Review - final review by group, with selected "intended audience"
 
+<<<<<<< HEAD
```

Comment:
There’s some merge conflict stuff that has remained here...

Reply:
Fixed this